tests/e2e/volumepropverification_test.go

```diff
@@ -43,6 +43,12 @@ var (
 	containerName    string
 )
 
+func TestMain(m *testing.M) {
```

Comment:
This is the current code, I'd removed it earlier but restoring the code as is. Let the author of this code refactor this file.